src/components/Chat.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable eqeqeq */
```

Comment:
Please don't disable lints without a good reason. Instead, you can use the `===` and `!==` operators, as ESLint suggests.